doozerlib/cli/scan_sources.py

```diff
@@ -109,6 +110,7 @@ def add_image_meta_change(meta, rebuild_hint: RebuildHint):
         oldest_image_event_ts = None
         newest_image_event_ts = 0
         for image_meta in runtime.image_metas():
+            dgk = image_meta.distgit_key
```

Comment:
`dkg` is used in line 167 and 170 for logging but we forgot to define it within the loop.
Maybe adding this variable here is not obvious because the actual use is far away from this line.
I will remove this and open another PR for the fix.